es-ES/readme.md

```diff
@@ -0,0 +1,703 @@
+# ![AVA](https://github.com/sindresorhus/ava/blob/master/media/header.png)
+
+> Test runner futurista
+
+[![Build Status: Linux](https://travis-ci.org/sindresorhus/ava.svg?branch=master)](https://travis-ci.org/sindresorhus/ava) [![Build status: Windows](https://ci.appveyor.com/api/projects/status/igogxrcmhhm085co/branch/master?svg=true)](https://ci.appveyor.com/project/sindresorhus/ava/branch/master) [![Coverage Status](https://coveralls.io/repos/sindresorhus/ava/badge.svg?branch=master&service=github)](https://coveralls.io/github/sindresorhus/ava?branch=master) [![Gitter](https://img.shields.io/badge/Gitter-Join_the_AVA_chat_%E2%86%92-00d06f.svg)](https://gitter.im/sindresorhus/ava)
+
+A pesar de que JavaScript se ejecuta en un solo hilo, IO en Node.js puede ejecutarse en paralelo debido a su naturaleza asíncrona. AVA se aprovecha de esto y corre sus tests al mismo tiempo, lo que es especialmente beneficioso para tests pesados en IO. Además, los archivos de test se ejecutan en paralelo como procesos separados, que le da un mejor rendimiento y un entorno aislado para cada archivo de test. [Cambiando](https://github.com/sindresorhus/pageres/commit/663be15acb3dd2eb0f71b1956ef28c2cd3fdeed0) de Mocha a AVA en Pageres llevó el tiempo de test por debajo de 31 segundos, concretamente a 11 segundos. El tener tests que se ejecutan al mismo tiempo nos obliga a escribir tests atómicos, es decir, los tests no dependen del estado global o el estado de otros tests, lo que está muy bien!
+
+*Lea nuestra [guía de contribución](contributing.md) si está pensando en contribuir (issues/PRs/etc).*
+
+
+## Tabla de contenidos
+
+- [Uso](#usage)
+- [Uso de CLI](#cli)
+- [Documentación](#documentation)
+- [API](#api)
+- [Aserciones](#assertions)
+- [FAQ](#faq)
+
+
+## ¿Por qué AVA?
+
+- Mínimo y rápido
+- Sintaxis simple de test
+- Ejecuta tests concurrentemente
+- Fuerza a escribir tests atómicos
+- No globales implícitos
+- [Entorno aislado para cada archivo de test](#isolated-environment)
```

Comment:
Bad link, replace `[Entorno aislado para cada archivo de test](#isolated-environment)` by `[Entorno aislado para cada archivo de test](#entorno-aislado)`

**It is necessary to check all the links**